packages/next-auth/src/core/init.ts

```diff
@@ -91,7 +91,6 @@ export async function init({
     // JWT options
     jwt: {
       secret, // Use application secret if no keys specified
-      maxAge, // same as session maxAge,
```

Comment:
Let's do `userOptions.jwt?.maxAge ?? maxAge`

Reply:
Agreed, I’ll get to this today